assets/data/cards_pt-br.json

```
@@ -63,6 +63,38 @@
         {
             "title": "Polimorfismo",
             "description": "O polimorfismo permite que objetos de diferentes tipos se comportem de maneira semelhante através de uma interface compartilhada."
+        },
+        {
+            "title": "Desenvolvimento Front-end",
+            "description": "Desenvolvimento web front-end é o desenvolvimento da interface gráfica do usuário de um site, através do uso de HTML, CSS e JavaScript, para que os usuários possam visualizar e interagir com esse site."
```

Alterei para uma linguagem neutra e como sugestão acho legal deixar algo mais genérico para se encaixar no front-wne mobile também, o que acha?

```suggestion
            "description": "Desenvolvimento front-end é o desenvolvimento da interface gráfica, para que as pessoas usuárias possam visualizar e interagir com o que foi desenvolvido."
```